### PR TITLE
BOAC-2277, add Student, by name or SID (with autosuggest), in 'New Note' form

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/BOAC-2277

Minor override options in `Autocomplete` component.

### CODE REVIEW

**Comment by paulkerschen** [comment]
@johncrossman Overrides make sense! One thing I learned in the course of writing this component is that `v-model` is just shorthand for a `value` property plus an `@input` listener. So I’m curious whether just adding `@input=“addStudent”` to your Autocomplete element would allow you to do without `studentModel` and the watcher.

**Comment by johncrossman** [comment]
@paulkerschen - Just experimented with your suggestion and it works. Amendment coming soon.

File: src/components/note/CreateNoteAddStudent.vue [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    </div>
+    <div class="mb-2">
+      <Autocomplete
+        id="create-note-add-student"
+        :key="resetAutoCompleteKey"
+        class="w-75"
+        :on-esc-form-input="onEscFormInput"
+        :show-add-button="true"
+        :source="findStudentsByNameOrSid"
+        @input="addStudent">
```
Comment (johncrossman) [inline src/components/note/CreateNoteAddStudent.vue]: Amended.

Comment (paulkerschen) [reply src/components/note/CreateNoteAddStudent.vue]: 👌

**Pull request merged**